lib/callbacks.ex

```diff
@@ -13,9 +13,16 @@ defmodule Callbacks do
     response = Requester.call(endpoint[:method], endpoint[:url], credentials)
     {:reply, response, credentials}
   end
+
   def handle_call({:get_user_jobs, username}, _from, credentials) do
     endpoint = Endpoints.Jobs.get_user_jobs(username)
     response = Requester.call(endpoint[:method], endpoint[:url], credentials)
     {:reply, response, credentials}
   end
+
+  def handle_call({:list_builds}, _from, credentials) do
```

Comment:
Let's decide if we want to use get or list.

Reply:
IMHO we should use :**get_builds**